modules/store/package.json

```diff
@@ -19,10 +19,13 @@
   "homepage": "https://github.com/ngrx/platform#readme",
   "peerDependencies": {
     "@angular/core": "NG_VERSION",
-    "rxjs": "RXJS_VERSION"
+    "rxjs": "RXJS_VERSION",
+    "@angular-devkit/core": "NG_DEVKIT_VERSION",
```

Comment:
Remove the devkit peer dependencies. If you're using the Angular CLI or the Schematics CLI, we can rely on those dependencies implicitly.

Reply:
Also, add the `NG_DEVKIT_VERSION` replaces to the `modules/schematics/package.json` and bump the version to `^0.6.0`

Reply:
Good to know, thanks!